data/locale/fr-FR.ini

```diff
@@ -30,8 +30,8 @@ NDIPlugin.OutputSettings.Main.Name="Nom NDI™"
 NDIPlugin.OutputSettings.Preview.Name="Nom NDI™"
 NDIPlugin.FilterName="Sortie NDI™ dédiée"
 NDIPlugin.AudioFilterName="Sortie NDI™ dédiée (Audio seul)"
-NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Correction du compositing alpha"
-NDIPlugin.LibError.Title="Runtime NDI™ introuvable"
-NDIPlugin.LibError.Message.Win="La bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur du runtime ici : <a href='http://new.tk/NDIRedistV5'>http://new.tk/NDIRedistV5</a>"
-NDIPlugin.LibError.Message.macOS="Le bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur du runtime ici : <a href='http://new.tk/NDIRedistV5Apple'>http://new.tk/NDIRedistV5Apple</a>"
-NDIPlugin.LibError.Message.Linux="La bibliothèque NDI™ est introuvable. Installez libndi sur votre système.<br>Vous pouvez en trouvez une copie sur la page GitHub d'obs-ndi."
+NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Correction de simulation de transparence"
+NDIPlugin.LibError.Title="Bibliothèque NDI™ introuvable"
+NDIPlugin.LibError.Message.Win="La bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur ici : <a href='http://new.tk/NDIRedistV5'>http://new.tk/NDIRedistV5</a>"
+NDIPlugin.LibError.Message.macOS="Le bibliothèque NDI™ est introuvable sur votre système.<br>Téléchargez l'installateur ici : <a href='http://new.tk/NDIRedistV5Apple'>http://new.tk/NDIRedistV5Apple</a>"
```

Comment:
Google Translate suggests:

`Exécution NDI™ introuvable.<br>Téléchargez le programme d'installation ici : <a href='http://new.tk/NDIRedistV5Apple'>http://new.tk/NDIRedistV5Apple</a>`

Other than the url itself, anything wrong w/ that?